### DISCUSSION TITLE
Infrastructure as code

### DISCUSSION THREAD

**Discussion by loftwah** [EddieHubCommunity/LinkFree#1444] [Ideas]
I like that Eddie has added the `YAML` files for deploying this on Kubernetes. I'm considering putting together a Terraform project that builds the required infrastructure, as well as deploying it on Kubernetes. It doesn't have to be Terraform, but that is my favourite to use, I could also do this with the AWS CDK or similar.

**Comment by eddiejaoude** [comment]
Moved to Vercel so I will close
